memelist.php

```diff
@@ -43,19 +43,19 @@
     . "Be like Bill.",
     "This is Bill.\n\n"
     . "Bill proofreads his Facebook posts\n"
-    . "before he sends them\n\n"
-    . "Bill is awesome.\n"
+    . "before he sends them.\n\n"
+    . "Bill is smart.\n"
     . "Be like Bill.",
     "This is Bill.\n\n"
-    . "Bill meets Della\n"
-    . "Bill loves Della\n"
-    . "Della loves Bill\n"
+    . "Bill meets Della.\n"
+    . "Bill loves Della.\n"
+    . "Della loves Bill.\n"
     . "Bill and Della dumps social networks.\n\n"
     . "Bill and Della are happy.\n"
     . "Be like Bill and Della.",
     "This is Bill.\n\n"
-    . "Bill isn't an online dating creep\n"
-    . "Bill likes the romantic way\n"
+    . "Bill isn't an online dating creep/\n"

```

Comment:
Please remove the triling `/`  `"Bill isn't an online dating creep/\n" --> `"Bill isn't an online dating creep.\n"
`